sale_validity/__manifest__.py

```diff
@@ -1,15 +1,16 @@
 # © 2013-2017 Camptocamp SA (Jacques-Etienne Baudoux)
 # © 2014-2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# Copyright 2024 Heliconia Solutions
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Sale Validity",
-    "version": "14.0.1.0.2",
+    "version": "16.0.1.0.0",
     "category": "Sales",
     "license": "AGPL-3",
     "summary": "Set a default validity delay on quotations",
     "depends": ["sale_management"],
-    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "author": "Camptocamp, Heliconia Solutions, Odoo Community Association (OCA)",
```

Comment:
@HeliconiaSolutions You cannot put yourself as author if no major contribution is added to the module. Thanks

Reply:
@rousseldenis Thank you for the clarification. Noted, we have made the necessary updates accordingly.